File: src/main/java/br/com/correios/enderecador/bean/GlobalBean.java
package br.com.correios.enderecador.bean;

public class GlobalBean {
    private static GlobalBean globalBean;

    private String mostraMensagem = "SIM";

    public static GlobalBean getInstance() {
        if (globalBean == null)
            globalBean = new GlobalBean();
        return globalBean;
    }

    public String getMostraMensagem() {
        return this.mostraMensagem;
    }

    public void setMostraMensagem(String mostraMensagem) {
        this.mostraMensagem = mostraMensagem;
    }
}

File: src/main/kotlin/br/com/correios/enderecador/bean/GlobalBean.kt
package br.com.correios.enderecador.bean

class GlobalBean {
    var mostraMensagem = "SIM"

    companion object {
        private var globalBean: GlobalBean? = null
        @JvmStatic
        val instance: GlobalBean?
            get() {
                if (globalBean == null) {
                    globalBean = GlobalBean()
                }
                return globalBean
            }
    }
}
